### PR TITLE
Expose kafka client id

### PR DESCRIPTION
**Pull request by PhoenixRion**
The team running the Kafka I was using required that I set the client id for reads and writes.  For reference, the default client id is "sarama"

### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [x] Associated README.md updated.
- [ ] Has appropriate unit tests.

I ran "make lint" and "make test", the latter showed failures for unrelated files. (outputs/socket_writer, outputs/influxdb, inputs/syslog, inputs/socket_listener, inputs/ping, inputs/http_response, inputs/fluentd)

### CODE REVIEW

File: plugins/inputs/kafka_consumer/kafka_consumer.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ import (
 
 type Kafka struct {
 	ConsumerGroup string
+	ClientId      string `toml:"client_id"`
```
Comment (danielnelson) [inline plugins/inputs/kafka_consumer/kafka_consumer.go]: Call the variable `ClientID`

File: plugins/inputs/kafka_consumer/README.md [hunk 2]
```diff
@@ -22,6 +22,9 @@ and use the old zookeeper connection method.
   ## Offset (must be either "oldest" or "newest")
   offset = "oldest"
 
+  ## Optional client id
+  # client_id = "my_client"
```
Comment (danielnelson) [inline plugins/inputs/kafka_consumer/README.md]: Let's have the example ID be `Telegraf`?  Perhaps this should be the default instead of "sarama"?

Comment (PhoenixRion) [reply plugins/inputs/kafka_consumer/README.md]: The default value of sarama is set by the underlying implementation used by both kafka plugins.  I can add the logic to set the id to "Telegraf" when a value is not supplied.

File: plugins/inputs/kafka_consumer/kafka_consumer.go (hunk 1 above)
Comment (PhoenixRion) [reply plugins/inputs/kafka_consumer/kafka_consumer.go]: 👍

**Comment by PhoenixRion** [comment]
ATM I am running the test on a Mac v10.13.2 (17C88)

**Comment by PhoenixRion** [comment]
https://github.com/influxdata/telegraf/issues/4420

**Comment by danielnelson** [comment]
Thank you!

**Pull request merged**